app/views/layouts/application.html.erb

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <% content_for :error_tracking do %>
```

Comment:
Why do we need to use a content tag here? does it have something to do with the `append_javascript_pack_tag` helper?

Reply:
You'll see it's the recommended approach here https://github.com/shakacode/shakapacker#view-helper-append_javascript_pack_tag-and-append_stylesheet_pack_tag

The alternative is we can move error tracking up into the head and render it there but `append_javascript_pack_tag` must come before `javascript_pack_tag`

Reply:
Gotcha, thanks for those links mate.